lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,6 +27,24 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instance(instance_name)
```

Comment:
`InstanceGroup` is a type of grouping of instances. You can use it do load balance and auto-scaling: https://cloud.google.com/compute/docs/instance-groups/